docs/guides/publisher-client-side.md

```diff
@@ -318,20 +328,20 @@ window.__uid2.callbacks.push(async (eventType, payload) => {
 </script>
 ```
 
-## Checking the Integration
+## Check that the Token Was Successfully Generated
 
 To check that the token was successfully generated, use the browser's developer tools to look for the token in local storage.
 
 ![Publisher Workflow](images/TokenDebugger.png)
 
-If there was a problem generating the token, find the request in the Network tab. You can find the request by filtering for the string client-generate. Information about why the request failed should be available in the response.
+If there was a problem generating the token, find the request in the **Network** tab. You can find the request by filtering for the string `client-generate`. Information about why the request failed should be available in the response.
 
 ![Publisher Workflow](images/NetworkTraffic.png)
 
 
-## Example codes: Hashing and Base64-encoding
+## Example codes: Hashing and Base64 Encoding
 
-The following codes are examples on how to generate email and phone hashes in JavaScript.
+The following codes are examples of how to generate email and phone hashes in JavaScript. {**GWH_SW_04 should I put this into the Normalization and Encoding article?**}
```

Comment:
_{**GWH_SW_04 should I put this into the Normalization and Encoding article?**}_

yes please and maybe add an explicit link to these js example codes in the "Configure the UID2 SDK for JavaScript" section above?

Reply:
@sunnywu Done (both of the above) thx.